main.go

```diff
@@ -450,6 +480,21 @@ func createDatabase(project, instance, dbName string, conv *internal.Conv, out *
 	return fmt.Sprintf("projects/%s/instances/%s/databases/%s", project, instance, dbName), nil
 }
 
+// getSQL return database type which is to be used for conversion.
+// Returns error if --driver flag is not set or database is not supported
+func getSqlType() (string, error) {
```

Comment:
Please add `driverName` as a function argument instead of using a global variable. The reason is that if I want to write a unit test for this function, it is hard to do it with a global variable. Thinking more about a `pure function`.

Reply:
`getSqlType` would not be used at any other place as mentioned in `ProcessDump()` comment. I will remove it.